themes/_core/js/product.js

```diff
@@ -132,3 +139,18 @@ $(document).ready(function () {
     });
   });
 });
+
+function getRequestParameter(preview) {
+  var vars = {};
+  window.location.href.replace(location.hash, '').replace(
+    /[?&]+([^=&]+)=?([^&]*)?/gi,
+    function (m, key, value) {
+      vars[key] = value !== undefined ? value : '';
+    }
+  );
+  if (preview) {
+    return vars[preview] ? vars[preview] : null;
+  }
+
+  return vars;
+}
```

Comment:
As this function is generic, I think it would be better to:
1. Rename it as `psGetRequestParameter` in order to avoid collisions.
2. Move it into common.js and import it here

Also, please don't forget to add a JSDoc header on new functions.